homeassistant/components/switcher_kis/climate.py

```diff
@@ -118,6 +119,10 @@ def __init__(
             if features["swing"] and not remote.separated_swing_command:
                 self._attr_supported_features |= ClimateEntityFeature.SWING_MODE
 
+        if len(self.hvac_modes) > 1:
+            self._attr_supported_features |= (
+                ClimateEntityFeature.TURN_OFF | ClimateEntityFeature.TURN_ON
+            )
```

Comment:
As I explained in my previous comment all Switcher devices support On/Off commands.
We first start with https://github.com/home-assistant/core/blob/f42a30ca523f464792b36612b3af610af8365c1a/homeassistant/components/switcher_kis/climate.py#L108

and search for the modes for ON, there is no possibility that there are no modes for on, we just don't know which modes at start.

You can simply set support for both on & off at init.

```python
        self._attr_supported_features = (
            ClimateEntityFeature.TURN_OFF | ClimateEntityFeature.TURN_ON
        )
```